lib/puppet/provider/rabbitmq_cli.rb

```diff
@@ -40,6 +40,10 @@ def self.rabbitmq_version
     output = rabbitmqctl('-q', 'status')
     version = output.match(%r{RabbitMQ.*?([\d\.]+)})
     @rabbitmq_version = version[1] if version
+    if @rabbitmq_version == nil
+      version = output.match /RabbitMQ version: ([\d\.]+)/
```

Comment:
could this be simplified into a single regex that matches both the old / new use cases?

Reply:
my suggestion is to start with a new test in https://github.com/voxpupuli/puppet-rabbitmq/blob/master/spec/unit/puppet/provider/rabbitmq_cli_spec.rb